v2/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/libp2p/go-libp2p-kad-dht/v2
 
-go 1.20
+go 1.21
+
+replace github.com/multiformats/go-multistream => github.com/multiformats/go-multistream v0.4.1
```

Comment:
Why do we need this replace? Note, `0.5.0` was released a week ago.

Reply:
Removed it. I recall I couldn't get the build to work without it originally, but it seems fine now